core/drivers/sub.mk

```diff
@@ -19,3 +19,4 @@ srcs-$(CFG_DRA7_RNG) += dra7_rng.c
 srcs-$(CFG_STIH_UART) += stih_asc.c
 srcs-$(CFG_ATMEL_UART) += atmel_uart.c
 srcs-$(CFG_MVEBU_UART) += mvebu_uart.c
+srcs-$(CFG_SUNXI_UART) += sunxi_uart.c
```

Comment:
Are we sure, we need new UART driver here?

For instance, in our pine64 port, we are re-using serial8250_uart driver (it looks pretty much
compatible with what allwinner has on most its SoC's).

Reply:
Nice catch. I've tried 8250 UART and it seems to me that it works well. 
But it doesn't support DT? I think I need to move the DT support part to there?